cmd/services.go

```diff
@@ -47,6 +53,66 @@ func init() {
 	cmds = append(cmds, appCmd)
 }
 
+func listCategoriesCmd(cliCtx *cli.Context) error {
+	client, err := api.New(api.Analytics, api.Catalog)
+	if err != nil {
+		return err
+	}
+
+	if err := maxOptionalArgsLength(cliCtx, 0); err != nil {
+		return err
+	}
+
+	var products []*models.Product
+	var providers []*models.Provider
+	var categories map[string][]*models.Product
+	var providerNames map[string]*models.Provider
+
+	categories = make(map[string][]*models.Product)
```

Comment:
you can do `categories := make(map[string][]*models.Product)` instead and remove the `var` declaration above.